docs/reference/index.md

```diff
@@ -1,40 +1,70 @@
 
-## Additional components [additional-components]
+### More
 
-APM Agents work in conjunction with the [APM Server](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md), [Elasticsearch](docs-content://get-started/index.md), and [Kibana](docs-content://get-started/the-stack.md). The [APM Guide](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md) provides details on how these components work together, and provides a matrix outlining [Agent and Server compatibility](docs-content://solutions/observability/apps/apm-agent-compatibility.md).
+The examples above highlight a couple of use cases that you can achieve with the agent and the {{stack}}. However, since the agent not only configures the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/) but also provides [direct access](manual-instrumentation.md) to its features, you can generate your own data in ways that best suit your needs and take advantage of {{stack}}'s tools, such as [creating alerts](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html) for when something interesting for you happens (maybe when an error is recorded), as well as [custom dashboards](https://www.elastic.co/guide/en/kibana/current/dashboard.html) to display your data the way you need to see it, and [much more](https://www.elastic.co/kibana/features).
 
+## Features
+
+On top of the features that come with the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/), the agent also adds the following ones.
+
+### Disk buffering
+
+Your application's telemetry data is stored locally before being sent to your {{stack}} and then removed once successfully exported, or to make room for new telemetry data, if needed. This minimizes the risk of data loss due to internet connectivity issues.
+
+### Real time
+
+For [distributed tracing](#distributed-tracing) to work properly, your application's time should be in sync with the [coordinated universal time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time). This is sometimes an issue for Android applications, as the time provided by the OS is often not accurate enough. The agent aims to synchronize telemetry timestamps with the universal time to ensure a reliable view of event timelines.
+
+### Session
+
+A session groups telemetry data within a time frame when your application is active, presumably to carry out a single task, operation, or goal at a given time, by adding a `session.id` attribute to all the spans and logs that come out of your application. This helps narrowing down the amount of data that you should take a look at for cases in which you'd want to investigate the steps that ended up in an error within your app, or just to know what are the common actions performed on each use.
```

Comment:
A reader might get lost in all the commas. 🙂 

```suggestion
A session groups telemetry data within a time frame when your application is active (presumably to carry out a single task, operation, or goal) by adding a `session.id` attribute to all the spans and logs that come out of your application. This helps limit the volume of data you need to look at when investigating the steps that led to an error within your app. It can also give you insight into the common actions performed on each use.
```